Source/WebKit/DerivedSources.make

```diff
@@ -885,6 +885,8 @@ EXTENSION_INTERFACES = \
     WebExtensionAPIPort \
     WebExtensionAPIRuntime \
     WebExtensionAPIScripting \
+    WebExtensionAPISidebarAction \
```

Comment:
nit: `WebExtensionAPISidePanel \` should technically come first since Side < Sidebar. the way i always double check ordering is copying the lines in question and running `pbpaste | sort | pbcopy` in terminal!